### PR TITLE
[ADD] connector_jira_tempo

### PR DESCRIPTION
**Pull request by simahawk**
early draft of this thing...

### CODE REVIEW

File: connector_jira_tempo/models/account_analytic_line/common.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    # def tempo_read_worklog(self, worklog):
+    #     url = self._tempo_get_url('worklogs')
+    #     r = self.client._session.get(url, params={'dateFrom': '2018-01-01'})
+    #     return {}
+
+    def tempo_read_approval(self, worklog):
+        username = worklog['author']['key']
+        url = self._tempo_get_url('timesheet-approval/current')
+        resp = self.client._session.get(url, params={'username': username})
+        return resp.json()
```
Comment (guewen) [inline connector_jira_tempo/models/account_analytic_line/common.py]: Here's what you can do by relying on the jira lib internals too: https://github.com/guewen/connector-jira/commit/338308a0c11b8ddb39931d7696dcfbfb4b7d561f

Comment (guewen) [reply connector_jira_tempo/models/account_analytic_line/common.py]: Hmm actually I think I can't pass the headers, to be continued.

Comment (guewen) [reply connector_jira_tempo/models/account_analytic_line/common.py]: This one works https://github.com/camptocamp/connector-jira/commit/e32996851ce719aa752fc54f7ba19bb60d63033f

**Comment by guewen** [comment]
I added a fixup to remove the change to text_field, now this correction is in #17.

**Comment by guewen** [comment]
Rebased and added a correction

**Comment by simahawk** [comment]
@guewen any other important change to do here? Otherwise can merge and improve/fix later.

**Pull request merged**